components/camel-mail/src/main/java/org/apache/camel/component/mail/MailConsumer.java

```diff
@@ -95,6 +95,25 @@ protected void doStop() throws Exception {
         super.doStop();
     }
 
```

Comment:
I meant only the '@return' part :-)

Reply:
Ok. I understood to put the comment in the adoc instead. I updated the commit. Please check :)

Reply:
My fault. It looks good. When you can please a note about the change in the adoc, not this comment :-)

Reply:
Sorry, I'm not familiar with the adoc. Do I need a editor for it? Where should I document my change? I could not see a suitable place in the mail-component.adoc.

Reply:
You just need to add a section here https://github.com/apache/camel/blob/master/components/camel-mail/src/main/docs/mail-component.adoc

By the way, no problem, we can do this later.

Reply:
Sorry I still don't get it :) What exactly you like me to document? The commit is just code optimization, right? :)

Reply:
Usually people notice performance optmizations, but sometime for special cases people may also note problems, so we need to add a note about what you've done between 3.0 and 3.1

Reply:
I updated the mail.component.adoc. Please have a look.